lib/rouge/all.rb

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+Rouge::LOAD_ALL = true
+
+require 'rouge'
+
+Dir.glob(lexer_dir('*rb')).each { |f| Rouge::Lexers.load_lexer(f.sub(lexer_dir, '')) }
```

Comment:
I think we can safely modify the string destructively here:
```suggestion
Dir.glob(lexer_dir('*rb')).each { |f| Rouge::Lexers.load_lexer(f.sub!(lexer_dir, '')) }
```

Reply:
This is just the code that's currently in `lib/rouge.rb`. If we're going to change it, that's fine but I don't want to do it as a result of this PR given they're not related.